plutus-tx-plugin/test/Budget/show.plc.golden

```diff
@@ -896,7 +896,14 @@
                                 (abs
                                   dead
                                   (type)
-                                  [ { Cons (con string) } (con string "a") ]
+                                  (lam
```

Comment:
Same eta-reduction problem as `monoConstructor.plc.golden`. This occurs multiple times in this test case.